cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -0,0 +1,172 @@
+
+
+@pytest.fixture(scope='function')
+def form_data_single_network(request, nvc_prov, rhvm_prov):
+    form_data = {'general': {'name': 'infra_map_{}'.format(fauxfactory.gen_alphanumeric()),
+                             'description': "Single Network migration  of VM from {} to {},".
+                             format(request.param[0], request.param[1])},
+                 'cluster': {'mappings': [_form_data_cluster_mapping(nvc_prov, rhvm_prov)]},
+                 'datastore': {'Cluster ({})'.format(rhvm_prov.data.get('clusters')[0]):
+                 {'mappings':
+                 [_form_data_datastore_mapping(nvc_prov, rhvm_prov,
+                  'nfs', 'nfs')]}},
+                 'network': {'Cluster ({})'.format(rhvm_prov.data.get('clusters')[0]): {'mappings':
+                 [_form_data_network_mapping(nvc_prov, rhvm_prov,
+                  request.param[0], request.param[1])]}}}
+    return form_data
+
+
+@pytest.fixture(scope='function')
+def form_data_dual_datastore(request, nvc_prov, rhvm_prov):
+    form_data = {'general': {'name': 'infra_map_{}'.format(fauxfactory.gen_alphanumeric()),
+                             'description': "Dual Datastore migration  of VM from {} to {},"
+                             "& from {} to {}".
+                             format(request.param[0][0], request.param[0][1], request.param[1][0],
+                                    request.param[1][1])},
+                 'cluster': {'mappings': [_form_data_cluster_mapping(nvc_prov, rhvm_prov)]},
+                 'datastore': {'Cluster ({})'.format(rhvm_prov.data.get('clusters')[0]):
+                 {'mappings':
+                 [_form_data_datastore_mapping(nvc_prov, rhvm_prov,
+                  request.param[0][0], request.param[0][1]),
+                 _form_data_datastore_mapping(nvc_prov, rhvm_prov,
+                  request.param[1][0], request.param[1][1])]}},
+                 'network': {'Cluster ({})'.format(rhvm_prov.data.get('clusters')[0]): {'mappings':
+                 [_form_data_network_mapping(nvc_prov, rhvm_prov,
+                  'VM Network', 'ovirtmgmt')]}}}
+    return form_data
+
+
+@pytest.mark.parametrize('form_data_single_datastore', [['nfs', 'nfs'],
+                            ['nfs', 'iscsi'], ['iscsi', 'iscsi']], indirect=True)
+def test_single_datastore_single_vm_mapping_crud(appliance, create_provider,
+                                            form_data_single_datastore):
+    infrastructure_mapping_collection = appliance.collections.infrastructure_mapping
+    mapping = infrastructure_mapping_collection.create(form_data_single_datastore)
+    view = navigate_to(infrastructure_mapping_collection, 'All')
+    assert mapping.name in view.infra_mapping_list.read()
+
+
+@pytest.mark.parametrize('form_data_single_network', [['VM Network', 'ovirtmgmt'],
+                            ['DPortGroup', 'ovirtmgmt']], indirect=True)
+def test_single_network_single_vm_mapping_crud(appliance, create_provider,
```

Comment:
**REQUIRED**
CRUD is create, read, update, delete. In this test you do only create.

Reply:
This was actually at my request, given what is currently supported in the framework. I wanted to see some valid tests of the fixtures/generation via the available create method. These CF entities don't block from provider removal, or make for any additional load on the provider. I believe its okay to not clean them up at this time, and the tests will be updated as the framework support is added for Update/Delete.

Reply:
Though I was expecting a TODO + comments here explaining that intention and need for test expansion... ;)

Reply:
@mshriver thanks for jumping in on this, I will add TODO. @quarckster thanks for review, updating my PR.